src/pygeoprocessing/geoprocessing.py

```diff
@@ -2722,10 +2751,23 @@ def transform_bounding_box(
     target_ref = osr.SpatialReference()
     target_ref.ImportFromWkt(target_ref_wkt)
 
-    transformer = osr.CoordinateTransformation(base_ref, target_ref)
+    # Create a coordinate transformation
+    # In GDAL 3.0 spatial references no longer ignore Geographic CRS 
+    # Axis Order and conform to Lat first, Lon Second. Transforms
+    # expect (lat, lon) order as opposed to the GIS friendly (lon, lat). 
+    # See https://trac.osgeo.org/gdal/wiki/rfc73_proj6_wkt2_srsbarn Axis order
+    # issues. SetAxisMappingStrategy(osr.OAMS_TRADITIONAL_GIS_ORDER) swaps 
+    # the axis order, which will use Lon,Lat order for Geographic CRS, 
+    # but otherwise leaves Projected CRS alone
+    base_ref.SetAxisMappingStrategy(osr_axis_mapping_strategy)
+    target_ref.SetAxisMappingStrategy(osr_axis_mapping_strategy)
+    
+    transformer = osr.CreateCoordinateTransformation(base_ref, target_ref)
 
     def _transform_point(point):
         """Transform an (x,y) point tuple from base_ref to target_ref."""
+        # NOTE: In GDAL 3, if osr.OAMS_TRADITIONAL_GIS_ORDER not set then
+        # TransformPoint expects Lat,Lon and NOT Lon,Lat for Geographic CRS. 
```

Comment:
This is only true if the projection system has the {2, 1} axis order like WGS84 has. Could you either remove this comment or point out something more general like "TransformPoint's axis order is dependent on the `SetAxisMappingStrategy` set above".

Reply:
removed in [779ec92](https://github.com/natcap/pygeoprocessing/pull/31/commits/779ec92c5b38e30880d6d986ccdb88ddf644cbf7)